src/sentry/discover/translation/mep_to_eap.py

```diff
@@ -225,18 +263,13 @@ def translate_columns(columns):
 
         translated_func, did_update = function_switcheroo(column)
         if did_update:
+            translated_func = add_equation_prefix_if_needed(translated_func, need_equation)
             translated_columns.append(translated_func)
             continue
 
-        raw_function = match.group("function")
-        arguments = fields.parse_arguments(raw_function, match.group("columns"))
-        translated_arguments = []
-
-        for argument in arguments:
-            translated_arguments.append(column_switcheroo(argument)[0])
-
-        new_arg = ",".join(translated_arguments)
-        translated_columns.append(f"{raw_function}({new_arg})")
+        new_function = switch_arguments(match)
```

Comment:
nit: if this is the only place using `switch_arguments` can we revert the change to extract it into a separate function and just keep the code here? It just flows and doesn't interrupt readability if it's contained here